tools/cldr-apps/js/webpack.config.js

```diff
@@ -1,17 +1,28 @@
-const path = require('path');
+const path = require("path");
+const { VueLoaderPlugin } = require("vue-loader");
```

Comment:
just curious, would `import` instead of `require` work here?

Reply:
@btangmu not sure